err-logging/logging.py

```diff
@@ -0,0 +1,39 @@
+from datetime import datetime
+import os
+from errbot import BotPlugin, botcmd
+
+
+log_root = "./logs"
```

Comment:
Hmm, I'd like the plugin to be agnostic to where errbot is keeping it's stuff. However, I think I may need to update the systemd files on quartermaster side to make sure it executes from quartermaster's homedir, so that relative paths make sense.